app/delegate-server/ClientServer/Server.purs

```diff
@@ -0,0 +1,86 @@
+module DelegateServer.ClientServer.Server
+  ( clientServer
+  ) where
+
+import Prelude
+
+import Data.Newtype (unwrap)
+import Data.Tuple.Nested ((/\))
+import DelegateServer.ClientServer.Handlers.MoveBid (moveBidHandler)
+import DelegateServer.ClientServer.Handlers.PlaceBid (placeBidHandler)
+import DelegateServer.HydraNodeApi.WebSocket (HydraNodeApiWebSocket)
+import DelegateServer.State (AppState, AppM, runApp)
+import Effect.Aff.Class (liftAff)
+import Effect.Class (liftEffect)
+import Effect.Console (log)
+import HTTPure
+  ( Headers
+  , Request
+  , Response
+  , ServerM
+  , created
+  , emptyResponse'
+  , header
+  , headers
+  , notFound
+  , serve
+  , toString
+  ) as HTTPure
+import HTTPure (Method(Options, Post), (!?), (!@))
+import HTTPure.Status (ok) as HTTPureStatus
+import URI.Port (toInt) as Port
+
+clientServer :: AppState -> HydraNodeApiWebSocket -> HTTPure.ServerM
```

Comment:
Let's call it `delegateServer`. Or is it a different thing? 😮

Reply:
Renamed it to `server` (https://github.com/mlabs-haskell/hydra-auction-offchain/pull/31/commits/fcb2fa4c89328a8996b7e3facdc8358e6b5767cd).
It was named `clientServer` because I thought it would be used exclusively by clients. However, since there will be an additional authenticated endpoint for delegates to sign a commit transaction, the old name becomes unsuitable.